csv2bufr/__init__.py

```diff
@@ -30,7 +30,7 @@
 import os.path
 from typing import Any, Iterator, Union
 from jsonpath_ng.ext import parser
-
+from copy import deepcopy
```

Comment:
nitpick/your call: we should organize/alphabetize dependencies, in the following order:

- standard library
- installed packages
- local modules

Example:

```python
from copy import deepcopy
import csv
from datetime import timezone, datetime
import hashlib
from io import StringIO, BytesIO
import json
import logging
import os.path
from typing import Any, Iterator, Union

from eccodes import (codes_bufr_new_from_samples,
                     codes_set_array, codes_set, codes_get_native_type,
                     codes_write, codes_release, codes_get,
                     CODES_MISSING_LONG, CODES_MISSING_DOUBLE,
                     codes_bufr_keys_iterator_new,
                     codes_bufr_keys_iterator_next,
                     codes_bufr_keys_iterator_get_name, CodesInternalError)
from jsonpath_ng.ext import parser
from jsonschema import validate
```

Reply:
changed.